scipy/sparse/tests/test_base.py

```diff
@@ -59,6 +59,26 @@ def assert_array_equal_dtype(x, y, **kwargs):
     assert_(x.dtype == y.dtype)
     assert_array_equal(x, y, **kwargs)
 
+ARRAY_BACKED_FORMATS = frozenset(['dok'])
```

Comment:
I guess this should be `NON_ARRAY_BACKED_FORMATS`

Reply:
Oops.